### PR TITLE
Remove deprecated endpoints containing _xpack.

### PR DESCRIPTION
**Pull request by jtibshirani**
The endpoints with `_xpack` in their path were deprecated in 7.x and can now be
removed. This commit removes deprecated endpoints for the following APIs:
* deprecation
* graph
* license
* monitoring
* rollup
* SQL
* watcher

Relates to #35958.

### CODE REVIEW

**Comment by jtibshirani** [comment]
@elasticmachine run elasticsearch-ci/packaging-sample
@elasticmachine run elasticsearch-ci/packaging-sample-matrix

**Comment by jtibshirani** [comment]
@elasticmachine run elasticsearch-ci/packaging-sample

**Comment by jtibshirani** [comment]
@elasticmachine run elasticsearch-ci/packaging-sample-matrix

File: docs/reference/migration/migrate_8_0.asciidoc [hunk 1]
```diff
@@ -50,6 +50,16 @@ Indices created in Elasticsearch 6.x or before will need to be reindexed with
...
 
 =========================================
+
+[float]
+==== REST endpoints containing `_xpack`
+
+In 7.0, we deprecated REST endpoints that contain `_xpack` in their path. These
+endpoints are now removed in 8.0. Each endpoint that was deprecated and removed
+is replaced with a new endpoint that does not contain `_xpack`. As an example,
+`/{index}/_xpack/graph/_explore` is replaced by `/{index}/graph/_explore`.
```
Comment (rjernst) [inline docs/reference/migration/migrate_8_0.asciidoc]: Isn't graph prefixed with an underscore, ie `_graph`?

**Review by rjernst — APPROVED** [review]
LGTM

File: docs/reference/migration/migrate_8_0.asciidoc (hunk 1 above)
Comment (jtibshirani) [reply docs/reference/migration/migrate_8_0.asciidoc]: Oops, thanks. Yes `_xpack` has been removed but also `graph/_explore` -> `_graph/explore`.

**Comment by jtibshirani** [comment]
@elasticmachine run elasticsearch-ci/bwc

**Pull request merged**